app/components/twilio-make-call.js

```diff
@@ -0,0 +1,71 @@
+import Ember from 'ember';
+import AjaxPromise from '../utils/ajax-promise';
+
+export default Ember.Component.extend({
+
+  mobile: "",
+  hidden: Ember.computed.empty("mobile"),
+
+  mobileWithPrefix: function() {
+    var mobile = this.get("mobile");
+    var prefix = mobile.indexOf("+852") === -1 ? "" : "";
+    return this.get('session.currentUser.id') + "#" + this.get("offerId") + "#" + prefix + mobile.replace(/ /g, "");
+  }.property("mobile"),
+
+  displayNumber: function() {
+    var num = this.get("mobile").replace(/\+852/, "");
+    return num.length > 4 ? num.substr(0, 4) + " " + num.substr(4) : num;
+  }.property("mobile"),
+
+  initTwilioDeviceBindings: function() {
+    var twilio_device, twilio_token;
+    twilio_token = Ember.$('#twilio_token').data('token');
+    twilio_device = Twilio.Device;
+    twilio_device.setup(twilio_token, {
+      debug: true
+    });
+
+    twilio_device.error(function() {
+      Twilio.Device.disconnectAll();
+      return Ember.$("#hangup").hide();
+    });
+
+    twilio_device.disconnect(function() {
+      return Ember.$("#hangup").hide();
+    });
+  },
+
+  actions: {
+
+    makeCall: function(){
+      var params = { "phone_number": Ember.$('#phoneNumber').data('mobile') };
```

Comment:
@swatijadhav is there a need to use jQuery to access the mobile rather than straight to the Ember component variables? Doesn't this work:

    this.get('mobileWithPrefix')

?? Thanks. If so, there are a few places in this file where I'd love to see jQuery disappear.